app/pages/5_Model_assumptions.py

```diff
@@ -0,0 +1,83 @@
+    'central solar thermal', 'decentral solar thermal',
+    'central solid biomass CHP',
+    'central water tank storage', 'decentral water tank storage',
+    'water tank charger', 'water tank discharger',
+
+    'battery inverter', 'battery storage',
+    'biogas upgrading', 'direct air capture', 'electricity distribution grid', 'electricity grid connection',
+    'electrolysis', 'fuel cell', 'central gas CHP', 'central gas boiler',
+    'decentral gas boiler', 'hydrogen storage underground', 'hydrogen storage tank type 1',
+    'HVDC inverter pair', 'helmeth', 'micro CHP', 'OCGT', 'offwind',
+    'onwind', 'PHS', 'ror', 'methanation', 'solar-rooftop',
+    'solar-utility', 'Steam methane reforming'
+]
+
+st.write(costs.loc[considered])
+
+st.write("FOM: Fixed operation and maintenance costs. Given as percentage of overnight costs per annum.")
+st.write("VOM: Variable operation and maintenance")
+
+st.header("Assumptions to distribute heat demands")
+
+url = 'https://raw.githubusercontent.com/PyPSA/pypsa-eur/v0.9.0/data/heat_load_profile_BDEW.csv'
+
+heat_demand = pd.read_csv(url, index_col=0)
+
+st.write("Deriving hourly space heating demand profiles is performed in three steps:")
+st.write(
+    "First, daily profiles are determined by applying the degree day approximation to each region. "
+    "This involves utilizing daily averaged ambient air temperature data, where we assume heating above 15°C."
+)
+st.markdown(
+    "Second, heat demands are distributed following average day patterns derived using the [demandlib](https://demandlib.readthedocs.io/en/latest/) package. "
+    "They discriminate between a normal weekday or a weekend, and between the residential and the services sector. See the graphics below."
+)
+
+#st.line_chart(heat_demand['residential space weekday'])
+# Create an interactive plot with Plotly Express
+fig = sp.make_subplots(
+    rows=1, cols=1,
+    #subplot_titles=['residential space weekday', 'residential space weekend', 'services space weekday', 'services space weekend']
+)
+
+fig.add_trace(go.Scatter(x=heat_demand.index, y=heat_demand["residential space weekday"], mode='lines', name=f'residential weekday'), row=1, col=1)
+fig.add_trace(go.Scatter(x=heat_demand.index, y=heat_demand["residential space weekend"], mode='lines', name=f'residential weekend'), row=1, col=1)
+fig.add_trace(go.Scatter(x=heat_demand.index, y=heat_demand["services space weekday"], mode='lines', name=f'services weekday'), row=1, col=1)
+fig.add_trace(go.Scatter(x=heat_demand.index, y=heat_demand["services space weekend"], mode='lines', name=f'services weekend'), row=1, col=1)
+
+
+fig.update_xaxes(title_text='number of hour in a day')
+fig.update_yaxes(title_text='heat demand [per unit]')
+
+# Display the interactive plot using Streamlit
+st.plotly_chart(fig)
+
+st.write("Third, the per unit time-series are scaled with the nationally reported heat demand per annum.")
+
+st.header("Assumed heating demands per annum")
```

Comment:
Let's remove it for now? I recognise that it may be thought as an outline for future work, but there is a considerable risk that we'll simply forget about it (saying from the experience...) :D